### DISCUSSION TITLE
Freeze Datagrid headers

### DISCUSSION THREAD

**Discussion by polonskyg** [Megabit/Blazorise#2291] [Q&A]
I need to freeze datagrid headers row. How can I do it?

I saw this: [https://stackoverflow.com/questions/63233158/how-do-you-add-sticky-headers-to-blazorise-datagrid](url) but doesn't work for me. I don't see any property built in for that.

Thanks in advance. Guillermo.

**Comment by stsrki** [comment]
The feature is in the works but it is proven to be more difficult than we anticipated. You can track the PR here https://github.com/stsrki/Blazorise/pull/2214

Currently, I'm experimenting with the JavaScript approach and we'll see how it goes.
